I.pass-tests/04-find-needle/find-needle.js

```diff
@@ -1,3 +1,10 @@
-function findNeedle(words) {}
+function findNeedle(words) {
+    let result;
```

Comment:
if you look at the test, you have two arguments passed, expecting two parameters.